localisation/ko/docs/00-setup.md

```diff
@@ -0,0 +1,351 @@
+# 00: 개발 환경
+
+이 단계에서는 워크샵을 위한 개발 환경을 설정합니다.
+
+## 전제 조건
+
+준비를 위해 [README](../README.md) 문서를 참조하세요.
+
+## 시작하기
+
+- [GitHub Codespaces 사용](#github-codespaces-사용)
+- [Visual Studio Code 사용](#visual-studio-code-사용)
+  - [PowerShell 설치 👉 Windows 사용자용](#powershell-설치--windows-사용자용)
+  - [git CLI 설치](#git-cli-설치)
+  - [GitHub CLI 설치](#github-cli-설치)
+  - [Docker Desktop 설치](#docker-desktop-설치)
+  - [Visual Studio Code 설치](#visual-studio-code-설치)
+  - [Visual Studio Code 시작](#visual-studio-code-시작)
+- [GitHub Copilot 에이전트 모드 확인](#github-copilot-에이전트-모드-확인)
+- [커스텀 지시사항 준비](#커스텀-지시사항-준비)
+- [제품 요구사항 문서(PRD) 분석 및 API 설계](#제품-요구사항-문서prd-분석-및-api-설계)
+
+## GitHub Codespaces 사용
+
+1. 아래 이미지를 보고 다음 순서로 클릭하여 따라하세요: ![the "code" button image](https://img.shields.io/badge/%3C%3E_Code-green) 👉 ![the "codespaces" button image](https://img.shields.io/badge/Codespaces-white) 👉 ![the "create codespace on main" button image](https://img.shields.io/badge/Create_codespace_on_main-green)을 클릭하여 새로운 GitHub Codespace 인스턴스를 생성하세요. 네트워크 상태에 따라 5-10분 정도 걸릴 수 있습니다.
+
+   ![Starting with GitHub Codespaces](./images/setup-01.png)
```

Comment:
This image path should point to the original one, which is `../../../docs/images/setup-01.png`.